### ISSUE TITLE
Netiface requirement for 0.10.4

### ISSUE THREAD

**Issue by mturilli** [radical-cybertools/radical.pilot#1814]
When installing RP from pip on MacOS 10.14 I see the following:
```
radical-pilot 0.50.22 has requirement netifaces==0.10.4, but you'll have netifaces 0.10.9 which is incompatible.
```
Local tests with EnTK perform as expected but I am not sure whether this would be a problem with more "realistic" runs. Is requirement for `==0.10.4` a hard requirement?

**Comment by mturilli** [comment]
BTW, should we add a topic:installation to our labels?

**Comment by andre-merzky** [comment]
10.0.4 is not a hard requirement anymore. This is already changed in the latest RU release, I'll make sure its getting fixed in RP, too.

**Comment by andre-merzky** [comment]
Re topic: yes, a topic `deployment` would be good (to also capture deployment during bootstrap etc).  I'll add it to all repos if you agree.

**Comment by mturilli** [comment]
Perfect!

**Comment by iparask** [comment]
@andre-merzky shouldn't we remove the requirement completely? It is satisfied by `radical.utils`

**Comment by andre-merzky** [comment]
@iparask : RP uses the module directly, not only via RP, so it is a dependency.  That RU pulls it does not matter - assume RP stops using the module and removes the dependency, that should not cause an RP install to fail.

**Comment by iparask** [comment]
I need to update the conda recipe then! Please tell me when this is going to be released so that I can fix the recipe

**Comment by andre-merzky** [comment]
The RP, RS and RU releases need to be synced this time, because of the saga-python -> radical.saga change.  This will not be completed this week, there is still a lot to do wrt. cleaning up, consistency and error checking.  If this is urgent I can do a hotfix on RP of course - let me know.

**Comment by andre-merzky** [comment]
This is scheduled to be released in v0.60.

**Comment by andre-merzky** [comment]
closed as  fixed.
